### DISCUSSION TITLE
town_block_limit didn't work

### DISCUSSION THREAD

**Discussion by manul694** [TownyAdvanced/Towny#6050] [Q&A] [answered]
I set town_block_limit to 1000, but townblock limit is infinity in some towns. How to fix it?
![image](https://user-images.githubusercontent.com/48861382/180262294-82d7655c-fa94-41fe-ac3e-8999dbca0ad4.png)

**Comment by LlmDl** [comment, marked as answer]
Set town_block_ratio to 0. If that doesn't work share you config so I can do more than guess at an answer.

**Reply by LlmDl** [reply]
The infinity symbol is caused by the town having infinite claims toggled on by an admin.

**Reply by manul694** [reply]
how to disable?

**Reply by LlmDl** [reply]
`/ta town NAME toggle unlimitedclaims`

**Reply by manul694** [reply]
Thanks, how to disable it to all towns?

**Reply by LlmDl** [reply]
Is it affecting all towns or only some towns?

**Reply by manul694** [reply]
it affecting all old towns, what created before today. New towns didn't have this

**Reply by LlmDl** [reply]
use the command I gave you on all the towns separately.

**Reply by manul694** [reply]
is make plugin with cycle which run this command for all towns - good idea?

**Reply by LlmDl** [reply]
I do not have the time to do that for you. I fix bugs, do things I find interesting and things for people that sponsor me. 

I will make you a plugin that does this for you for my standard rate of $25/hour.
